### PR TITLE
Damns Devils Gamemode to Hell

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Hell called, and it wanted its souls back and it's come to claim them, for this gamemode is well due for the blazing inferno.

Devils game mode is.....incredibly flawed. It just doesn't work in a low RP or even medium RP environment---ironically, it *might* work on a place like Baystation, but even then, I question it.

On top of this, the code for this mode is just...not great at all. 

Your soul has been found wanting, Devils, and I condemn you to eternal damnation.

The Codex Gigas, toy code gigas, and employment contracts are all sticking around; these either fit well, thematically, just as an aesthetic, or they provide a humorous/IC reason to exist.

:cl: Fox McCloud
del: removes Devils gamemode
/:cl:

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -55,10 +55,6 @@
 	var/antag_hud_icon_state = null //this mind's ANTAG_HUD should have this icon_state
 	var/datum/atom_hud/antag/antag_hud = null //this mind's antag HUD
 	var/datum/mindslaves/som //stands for slave or master...hush..
-	var/datum/devilinfo/devilinfo //Information about the devil, if any.
```
Comment (AffectedArc07) [inline code/datums/mind.dm:58]: inb4 this is why minds never GC

File: code/game/machinery/cloning.dm [hunk 2]
```diff
@@ -225,11 +219,6 @@ GLOBAL_LIST_INIT(cloner_biomass_items, list(\
 		return 0
 	if(clonemind.current && clonemind.current.stat != DEAD)	//mind is associated with a non-dead body
 		return 0
-	if(clonemind.damnation_type)
-		spooky_devil_flavor()
-		return 0
-	if(!clonemind.is_revivable()) //Other reasons for being unrevivable
```
Comment (AffectedArc07) [inline code/game/machinery/cloning.dm:231]: At first I was gonna ask about this
```dm
/datum/mind/proc/is_revivable() //Note, this ONLY checks the mind.
	if(damnation_type)
		return FALSE
	return TRUE
```
Then I read that

**Review by AffectedArc07 — APPROVED** [review]
Purge it all in the fires of hell

**Comment by kenken244** [comment]
I propose removing employment contacts too. The whole "nanotrasen owns your soul" and "you are a slave" thing is a bad meme and had no place on a server with actual rp

**Comment by Aquilar** [comment]
good riddens.

**Comment by MrMagolor** [comment]
> I propose removing employment contacts too. The whole "nanotrasen owns your soul" and "you are a slave" thing is a bad meme and had no place on a server with actual rp

Have you ever heard of a person leaving nanotrasen... alive?

**Comment by Carthusia** [comment]
> I propose removing employment contacts too. The whole "nanotrasen owns your soul" and "you are a slave" thing is a bad meme and had no place on a server with actual rp

God yes, or rewrite them to be less of a complete meme that ruins the atmosphere of any discussion where they need to be brought into an RP situation in any serious capacity.

**Comment by Cocacolagua** [comment]
can you still write whatever in the codex gigas to get meme instructions to defeat a "devil"?

**Comment by Spacemanspark** [comment]
@MrMagolor I mean, one of my characters plays as a sub-contracted member from Bishop, basically providing free labor to NT.

I feel removing the employment contracts or rewording them would be best. Our lore is very vague and while I wish we had a bit more emphasis on it, it does allow a lot of creative freedom for character design.

**Comment by AffectedArc07** [comment]
> I propose removing employment contacts too. The whole "nanotrasen owns your soul" and "you are a slave" thing is a bad meme and had no place on a server with actual rp

Reminder that this is a corporation with an on-station nuclear device, many historical incidents about employee mistreatment, deathsquads on standby, and more.

**Comment by Mitchs98** [comment]
> I propose removing employment contacts too. The whole "nanotrasen owns your soul" and "you are a slave" thing is a bad meme and had no place on a server with actual rp

100% agree. It's a really stale meme that I honestly don't think fits the server lore. Employee's being named as literal owned slaves is meh for any sort of RP. And as a whole basically everyone ignores it that isn't trying to buy into the stale meme of 'NT owns your soul boi'.

**Review by Kyep — APPROVED** [review]
I'll sign off on this as a head.
While the devil game mode in theory is neat... in practice its buggy as hell, unmaintained and just doesn't work well. It has never worked, and because nobody is updating it, I never expect it to work in future.

Heads up contributors: only add a major feature to the codebase if you are also willing to handle patching it, fixing bugs with it, etc for at least the next 3-12 months after it is added. Adding stuff that doesn't work and then not patching it will get your stuff removed... just like devils.

**Comment by Spacemanspark** [comment]
@AffectedArc07
Having nukes on a station is actually pretty commonplace in a lot of Sci fi fields, and in the world of parass13 there are many well known, documented threats that would give applicable reason to have such a device on the station. It's not really kept a secret from the crew, either.

As for mistreatment/deathsquad--to an extent, yes, I think those are good traits for the setting, but I disagree that it should mean we restrict players to a silly meme degree.

**Comment by Denthamos** [comment]
Good in paper, but in practice relies on people actually willing to be able to engage and RP with you and not just instantly toolbox your head in as soon as you mention a contract or anything of a sort.  Back to the shadowrealm with you devils.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**